### PR TITLE
Fix for add to collection visibility issue

### PR DESCRIPTION
**Pull request by therobbiedavis**
# Fixed
- Fixed: Fixed a bug where users couldn't select a collection when they had a lot of them.

### CODE REVIEW

File: UI/Web/src/app/cards/_modals/bulk-add-to-collection/bulk-add-to-collection.component.html [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
-</div>
-<form style="width: 100%" [formGroup]="listForm">
-<div class="modal-body">
-    <div class="form-group" *ngIf="lists.length >= 5">
-        <label for="filter">Filter</label>
-        <div class="input-group">
-            <input id="filter" autocomplete="off" class="form-control" formControlName="filterQuery" type="text" aria-describedby="reset-input">
-            <div class="input-group-append">
-                <button class="btn btn-outline-secondary" type="button" id="reset-input" (click)="listForm.get('filterQuery')?.setValue('');">Clear</button>
+<div id="add-to-collection">
```
Comment (majora2007) [inline UI/Web/src/app/cards/_modals/bulk-add-to-collection/bulk-add-to-collection.component.html:2]: Don't use ids

Comment (majora2007) [reply UI/Web/src/app/cards/_modals/bulk-add-to-collection/bulk-add-to-collection.component.html:2]: In addition, modals have a fixed structure, keep the structure as stated, don't wrap with a div. See:
https://ng-bootstrap.github.io/#/components/modal/examples

**Review by majora2007 — CHANGES_REQUESTED** [review]
Please try to solve this with the options already provided with ng-bootstrap's modal rather than rolling your own. 

Please test this on all devices plus other screens that have this type of thing. I'm pretty sure we just need to pass scrollable to the code that opens the modal.

File: UI/Web/src/app/cards/_modals/bulk-add-to-collection/bulk-add-to-collection.component.html (hunk 1 above)
Comment (therobbiedavis) [reply UI/Web/src/app/cards/_modals/bulk-add-to-collection/bulk-add-to-collection.component.html]: ID removed in abe0d9f

**Comment by therobbiedavis** [comment]
> Please try to solve this with the options already provided with ng-bootstrap's modal rather than rolling your own.
> 
> Please test this on all devices plus other screens that have this type of thing. I'm pretty sure we just need to pass scrollable to the code that opens the modal.

Tried passing scrollable, however there is some css being added in the head that I can not find.
image

I reviewed the bootstrap modal documentation and found that `encapsulation: ViewEncapsulation.None,` needed to be added to the component in order for the styles to pick up. Please see the testing verification below:

Desktop:
Screenshot 2022-01-26 210209

Mobile:
unnamed

https://user-images.githubusercontent.com/7907265/151279265-1d96c476-065d-4282-85ee-064fc7fe20b5.mp4

**Review by majora2007 — APPROVED** [review]
Looks good and feels good, let's merge

**Pull request merged**